src/components/Hero/index.tsx

```diff
@@ -23,11 +30,11 @@ export const Hero = () => {
             <i className="fas fa-rocket" /> Try Now!
           </Link>
           <a
-            href="https://github.com/kubeshop/botkube"
-            title="GitHub repository"
+            href="https://join.botkube.io"
+            title="Community Slack"
             className="button button--lg button--secondary"
           >
-            <i className="fab fa-github" /> View Source
+            <i className="fab fa-slack" /> Community Slack
```

Comment:
Do we need this button if there's the same link on the top? 🤔 It would be great to have 2 buttons inside the "hero" section to not "bloat" it. What do you think?